common/tor.c

```diff
@@ -230,8 +199,10 @@ static struct io_plan *tor_conn_init(struct io_conn *conn,
 	static struct tor_service_reaching reach;
```

Comment:
Better to use a `tal`-allocated `tor_service_reaching` than a statically-allocated one.  Use `ld` as parent.